docs/references/config/config.toml.md

```diff
@@ -0,0 +1,2058 @@
+```toml
+max_open_connections = 900
+```
+
+| Value type          | integer |
+|:--------------------|:--------|
+| **Possible values** | &gt; 0  |
+
+If you want to accept a larger number of connections than the default 900, make sure that you increase the maximum
+number of open connections in the operating system. Usually, the `ulimit` command can help with that.
+
+This value can be estimated by the following calculation:
+```
+$(ulimit -Sn) - {p2p.max_num_inbound_peers} - {p2p.max_num_outbound_peers} - {number of WAL, DB and other open files}
+```
+
+Estimating the number of WAL, DB and other files at `50`, and using the default soft limit of Debian Linux (`1024`):
+```
+1024 - 40 - 10 - 50 = 924 (~900)
+```
+
+Note, that macOS has a default soft limit of `256`. Make sure you calculate this value for the operating system CometBFT
+runs on.
+
+### rpc.max_subscription_clients
+Maximum number of unique clientIDs that can subscribe to events at the `/subscribe` RPC endpoint.
+```toml
+max_subscription_clients = 100
+```
+
+| Value type          | integer |
+|:--------------------|:--------|
+| **Possible values** | &gt;= 0 |
+
+### rpc.max_subscriptions_per_client
+Maximum number of unique queries a given client can subscribe to at the `/subscribe` RPC endpoint.
+```toml
+max_subscriptions_per_client = 5
+```
+
+| Value type          | integer |
+|:--------------------|:--------|
+| **Possible values** | &gt;= 0 |
+
+### rpc.experimental_subscription_buffer_size
+Experimental parameter to specify the maximum number of events a node will buffer, per subscription, before returning
+an error and closing the subscription.
+```toml
+experimental_subscription_buffer_size = 200
+```
+
+| Value type          | integer   |
+|:--------------------|:----------|
+| **Possible values** | &gt;= 100 |
+
+Higher values will accommodate higher event throughput rates (and will use more memory).
+
+### rpc.experimental_websocket_write_buffer_size
+Experimental parameter to specify the maximum number of events that can be buffered per WebSocket client.
+```toml
+experimental_websocket_write_buffer_size = 200
+```
+
+| Value type          | integer                                         |
+|:--------------------|:------------------------------------------------|
+| **Possible values** | &gt;= rpc.experimental_subscription_buffer_size |
+
+If clients cannot read from the WebSocket endpoint fast enough, they will be disconnected, so increasing this parameter
+may reduce the chances of them being disconnected (but will cause the node to use more memory).
+
+If set lower than `rpc.experimental_subscription_buffer_size`, connections could be dropped unnecessarily. This value
+should ideally be somewhat higher to accommodate non-subscription-related RPC responses.
+
+### rpc.experimental_close_on_slow_client
+Close the WebSocket client in case it cannot read events fast enough. Allows greater predictability in subscription
```

Comment:
```suggestion
> EXPERIMENTAL parameter!

Close the WebSocket client in case it cannot read events fast enough. Allows greater predictability in subscription
```